docs/user-guide/solver/03-outputs.md

```diff
@@ -262,4 +262,14 @@ Here is a list of new output variables in recent versions:
 | 8.6     | STS inj by plant, STS withdrawal by plant, STS lvl by plant                                                                                                                                                                                                                                                                                                                                                                                                                                       | details-STstorage-*.txt   | yes                |
-| 8.6     | CO2 EMIS., NH3 EMIS., SO2 EMIS., NOX EMIS., PM2_5 EMIS., PM5 EMIS., PM10 EMIS., NMVOC EMIS., OP1 EMIS., OP2 EMIS., OP3 EMIS., OP4 EMIS., OP5 EMIS.                                                                                                                                                                                                                                                                                                                                                | values-*.txt              | yes                |
\ No newline at end of file
+| 8.6     | CO2 EMIS., NH3 EMIS., SO2 EMIS., NOX EMIS., PM2_5 EMIS., PM5 EMIS., PM10 EMIS., NMVOC EMIS., OP1 EMIS., OP2 EMIS., OP3 EMIS., OP4 EMIS., OP5 EMIS.                                                                                                                                                                                                                                                                                                                                                | values-*.txt              | yes                |
+| 8.8 | PriceCSR (also introduced in 9.2) | values-*.txt | yes |
+| 8.8 | UNSP. ENRG CSR (also introduced in 9.2) | values-*.txt | yes |
+| 8.8 | LOLD_CSR, LOLP_CSR (also introduced in 9.2) | values-*.txt | yes |
+| 8.8 | MAX MRG CSR (also introduced in 9.2) | values-*.txt | yes |
+| 8.8 | OV. COST CSR (also introduced in 9.2) | values-*.txt | yes |
+| 9.1 | **Short-term storage** - dynamic groups instead of static groups. For any group :<br>\<STS group\>_injection <br> \<STS group\>_withdrawal <br> \<STS group\>_level| values-*.txt | yes |
+| 9.2 | MIN DTG by plant <br> PriceCSR (also introduced in 8.8) <br> UNSP. ENRG CSR (also introduced in 8.8) | details-\*.txt <br> values-\*.txt <br> values-*.txt| yes <br> yes <br> yes|
+| 9.2 | LOLD_CSR, LOLP_CSR (also introduced in 8.8) | values-\*.txt| yes |
+| 9.2 | MAX MRG CSR (also introduced in 8.8) | values-*.txt | yes |
+| 9.2 | OV. COST CSR (also introduced in 8.8) | values-*.txt | yes |
```

Comment:
Please just mention the oldest version with the feature. If the feature is present in branch 8.8, then it is also present in branch 9.2.

Reply:
What if a var was introduced on 8.8 and 9.2 ? 
Mentioning these 2 versions tells that this var does not belong to 9.0 and 9.1